docs/standard/net-standard.md

```diff
@@ -28,17 +28,9 @@ To find the highest version of .NET Standard that you can target, do the followi
 
 ### Which .NET Standard version to target
 
-When choosing a .NET Standard version to target, consider this trade-off:
+We recommend you target .NET Standard 2.0, unless you need to support an earlier version. Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0 is supported by all modern platforms and is the recommended way to support multiple platforms with one target.
 
-- The higher the version, the more APIs are available to your library's code.
```

Comment:
These two bullets seem useful. Do you not think we should keep them?

Reply:
I was thinking they are more misleading than helpful, as it's only in exceptional cases that you should consider 1.x anymore.